adcm_aio_client/mapping/_objects.py

```diff
@@ -151,16 +215,34 @@ def _to_payload(self: Self) -> list[dict]:
 
 
 class ClusterMapping(ActionMapping):
+    """
+    Node responsible for accessing mapping between `Component` objects and `Host` objects of specific cluster
```

Comment:
it's main purpose is to change mapping, not working with hosts and components (it's a pleasing side effect 🐱 )